docs/metrics-yaml-format.md

```diff
@@ -279,3 +285,17 @@ observation:
     objectName: solr/${core}:type=searcher,*
 
 ```
+
+## How to group metrics in YAML file
+Typically multiple metrics are grouped under single YAML file. 
+
+* DB data source - all metrics from single DB query can be grouped under one YAML file. For example, in Clickhouse 
```

Comment:
It is similar here as what I wrote for ES - e.g. for MySQL monitoring there are just 2 queries providing all metrics, but we split observations into 10-15 separate YAML files (again, separating them logically). And again worth mentioning is caching - Agent ensures that during single collection run query is run just once.